### DISCUSSION TITLE
Set user password

### DISCUSSION THREAD

**Discussion by aghaeian** [netdata/netdata#18879] [Q&A]
Hello,

How i can set user and password for restrict access view panel?

Best regards.

**Comment by ilyam8** [comment]
Hi, @aghaeian. There are a few options:

- [Use a web server to provide authentication](https://learn.netdata.cloud/docs/netdata-agent/securing-netdata-agents/#use-an-authenticating-web-server-in-proxy-mode).
- [Disable local UI and use NC](https://learn.netdata.cloud/docs/netdata-agent/securing-netdata-agents/#disable-the-local-dashboard).
